packages/grafana-ui/src/components/Cascader/Cascader.tsx

```diff
@@ -94,6 +93,8 @@ export class Cascader extends React.PureComponent<CascaderProps, CascaderState>
     return selectOptions;
   };
 
+  getSearchableOptions = memoizeOne((options: CascaderOption[]) => this.flattenOptions(options));
```

Comment:
I think `flattenOptions` should just be made into a regular function in this file, rather than a method on the class, but this is just a minor nit/personal preference so I can't put too much weight behind it.

Reply:
That makes sense - it looks like if we extract `flattenOptions` out, we will additionally need to pass in the separator too so I'll leave this here for now and then it can be updated when we change this to a functional component 👍